Phase#Beta v.4/css/indotuber.css

```diff
@@ -0,0 +1,414 @@
+}
+
+.lh-15{
+    line-height: 1.5;    
+}
+.mt-10{
+    margin-top: 10px;
+}
+.mt-0{
+    margin-top:0;
+}
+.pt-0{
+    padding-top:0;
+}
+.pt-0{
+    padding-top:0;
+}
+.mt-15{
+    margin-top: 15px;
+}
+.mt-20{
+    margin-top: 20px;
+}
+.mr-5{
+    margin-right: 5px;
+}
+.mr-10{
+    margin-right: 10px;
+}
+.ml-10{
+    margin-left: 10px;
+}
+.ml-5    {
+    margin-left: 5    px;
+}
+.mb-30{
+    margin-bottom: 30px;
+}
+.mb-10{
+    margin-bottom: 10px;
+}
+.mb-20{
+    margin-bottom: 20px;
+}
+.mb-5{
+    margin-bottom: 5px;
+}
+.mlr-0{
+    margin-left: 0;
+    margin-right: 0;
+}
+.mtb-0{
+    margin-top: 0;
+    margin-bottom: 0;
+}
+.fs-20{
+    font-size: 20px;
+}
+.fs-15{
+    font-size: 15px;
+}
+.fs-14{
+    font-size: 14px;
+}
+.fs-13{
+    font-size: 13px;
+}
+.plr-0{
+    padding-left: 0;
+    padding-right: 0;
+}
+.plr-5{
+    padding-left: 5px;
+    padding-right: 5px;
+}
+.plr-10{
+    padding-left: 10px;
+    padding-right: 10px;
+}
+.pd-5{
+    padding: 5px;
+}
+.pd-10{
+    padding: 10px;
+}
+.pd-15{
+    padding: 15px;
+}
+.pd-20{
+    padding: 20px;
+}
+
+/******************************
+          Navigation
+******************************/
+.topnav {
+    font-size: 14px;
+    padding: 5px;
+}
+
+.navbar-idtuber{
+    background-color: white;
+    border-bottom: 1px solid #eee;
+}
+
+.navbar-idtuber__logo{
+    width: 160px;
+    height: 40px;
+    float: left;
+    text-indent: -9999px;
+    margin-top: -10px;
+    background-image: url('../img/idtuber-icon.svg');
+    background-repeat: no-repeat;
+}
+.navbar .nav button{
+    display: block;
+    position: relative;
+    margin: 8px;
+    margin-left: 10px;
+}
+
+.navbar-toggle{
+    border: 1px solid #333!important;
+}
+
+.navbar-idtuber .navbar-toggle:hover{
+    background: #f5f5f5;
+}
+
+.navbar-idtuber .navbar-toggle .icon-bar {
+    background-color: #333;
+}
+
+.navbar-idtuber li a{
+    color: #727272;
+    margin-top: 7px;
+    padding: 8px;
+    margin-left: 10px;
+    -webkit-transition: all 0.3s;
+    -moz-transition: all 0.3s;
+    transition: all 0.3s;
+}
+
+.navbar-idtuber li a:hover{
+    color: #333;
+}
+
+@media(max-width:992px){
+    .navbar-idtuber__logo{
+        width: 120px;
+        height: 30px;
+        margin-top: -5px;
+    }
+}
+
+@media(max-width:767px){
+   .navbar .nav button{
+        width: 85%;
+        margin: 0 auto;
+        text-align: center;
+        margin-bottom: 10px;
+    }
+   
+   .navbar .nav li a{
+        width: 85%;
+        margin: 0 auto;
+        text-align: center;
+        margin-bottom: 5px;
+       border-radius: 5px;
+    }
+}
+
+/******************************
+        EOF Navigation
+******************************/
+
+
+
+
+
+
+/******************************
+        Content Global
+******************************/
+
+.content{
+    padding-top: 90px;
+    padding-bottom: 50px;
+}
+
+/*Right Panel*/
+.content__video-title{
```

Comment:
To tolong dipisah dong CSS buat header, footer sama random page.
Misal header sama footer ada di main.css
Terus isi random page ada di random.css